docs/user/installation.rst

```diff
@@ -89,3 +89,31 @@ Install Python modules
 .. code-block:: bash
 
   conda env create -f environment.yml
+  
+GPU Support
+===========
+
+To enable GPU support, you need to run express installation script with :code:`--gpu`. This options installs GPU-supported Tensorflow and necessary modules needed by Theano.
```

Comment:
Some small corrections:
- "... you need to run **the** express installation with **the argument** :code:'--gpu'"
- ".. and ~~necessary~~ modules needed by Theano." (I think necessary is redundant here).